iceoryx_utils/include/iceoryx_utils/internal/cxx/convert.inl

```diff
@@ -193,7 +193,7 @@ inline bool convert::fromString<uint64_t>(const char* v, uint64_t& dest)
         return false;
     }
 
-    auto call = makeSmartC(strtoull, ReturnMode::PRE_DEFINED_ERROR_CODE, {ULLONG_MAX}, {}, v, nullptr, 10u);
+    auto call = makeSmartC(strtoull, ReturnMode::PRE_DEFINED_ERROR_CODE, {ULLONG_MAX}, {}, v, nullptr, STRTOULL_BASE);
```

Comment:
``STRTOULL_BASE`` should not be in the cxx namespace if it is only used here. Make it internal to convert.